### DISCUSSION TITLE
helper to find an icon

### DISCUSSION THREAD

**Discussion by arallsopp** [Mariusthvdb/custom-icons#28] [Show and tell]
Howdo.

Having been digesting the helper I created for you, I've added something to hass-hue-icons that might be interesting.
Take a look at: https://arallsopp.github.io/hass-hue-icons/docs/build/tester/index.html

Its intended to let prospective users hunt for icons they want, via keyword as well as name.

Try 'spot' for example.

Feel free to reuse with accreditation.

**Comment by arallsopp** [comment]
I've just added another version of the tool here: https://arallsopp.github.io/hass-hue-icons/docs/build/tester/iconfinder.html

Layout is a bit squiffy on non mobile, but its getting there:
<img width="483" alt="image" src="https://user-images.githubusercontent.com/2462104/170228819-35471367-ef61-42cf-9738-68d43bb9f6c6.png">

**Comment by arallsopp** [comment]
Updated it, now you can point it at any icon map you like.

You just need to know a location for a webhosted script, and the name of the map they're using.

eg: https://arallsopp.github.io/hass-hue-icons/docs/build/tester/iconfinder.html?library=https://mariusthvdb.github.io/custom-icons/custom-icons.js&map=CUSTOM_ICONS_MAP will link to yours.

**Reply by Mariusthvdb** [reply]
cool! thanks Andy!

now need to find some time to add it to my repo, and the readme ofc...

**Reply by arallsopp** [reply]
Fair :)

You can probably just link to it like the [URL above](https://arallsopp.github.io/hass-hue-icons/docs/build/tester/iconfinder.html?library=https://mariusthvdb.github.io/custom-icons/custom-icons.js&map=CUSTOM_ICONS_MAP) and benefit from knowing it'll improve as we go. 

One thing I did notice, btw, your keywords sometimes duplicate the icon name. The HA picker seems to look for the words you type in the name and keyword properties, so there's no need for a keyword if its already present in the name, eg: [shutter and/or iphone](https://arallsopp.github.io/hass-hue-icons/docs/build/tester/iconfinder.html?library=https%3A%2F%2Fmariusthvdb.github.io%2Fcustom-icons%2Fcustom-icons.js&map=CUSTOM_ICONS_MAP&search=shutter+iphone).

**Reply by Mariusthvdb** [reply]
smart.... Ill check those, and correct
